schema.json

```diff
@@ -83,6 +83,38 @@
         "package": {
           "description": "Use this property to set the application ID of the APK. The value is set to `applicationId` in `build.gradle`.",
           "type": "string"
+        },
+        "signingConfigs": {
+          "description": "Use this to set the signing configurations for the app",
+          "type": "object",
+          "debug": {
+            "keyAlias": {
+              "type": "string"
+            },
+            "keyPassword": {
+              "type": "string"
+            },
+            "storeFile": {
+              "type": "string"
+            },
+            "storePassword": {
+              "type": "string"
+            }
+          },
+          "release": {
+            "keyAlias": {
+              "type": "string"
+            },
+            "keyPassword": {
+              "type": "string"
+            },
+            "storeFile": {
+              "type": "string"
+            },
+            "storePassword": {
+              "type": "string"
+            }
+          }
```

Comment:
I think we need to wrap the properties inside a `properties` property 😄 
```suggestion
          "properties": {
            "debug": {
              "type": "object",
              "properties": {
                "keyAlias": {
                  "type": "string"
                },
                "keyPassword": {
                  "type": "string"
                },
                "storeFile": {
                  "type": "string"
                },
                "storePassword": {
                  "type": "string"
                }
              },
              "required": ["storeFile"]
            },
            "release": {
              "type": "object",
              "properties": {
                "keyAlias": {
                  "type": "string"
                },
                "keyPassword": {
                  "type": "string"
                },
                "storeFile": {
                  "type": "string"
                },
                "storePassword": {
                  "type": "string"
                }
              },
              "required": ["storeFile"]
            }
          }
```
It would also be great to add the `description` field to each of these.

You can also use `yarn codegen` to verify this file.

Reply:
Good point!